packages/tatum-doge/src/constants.ts

```diff
@@ -0,0 +1,24 @@
+export const DOGE_DERIVATION_PATH = 'm/44\'/3\'/0\'/0'
+export const TESTNET_DERIVATION_PATH = 'm/44\'/1\'/0\'/0'
```

Comment:
This constant already exists in tatum-core. Please remove it and import directly from the core lib.

Reply:
it is commented in core

Reply:
Ok, can you uncomment it, please? Because it should work and it's my mistake that it's commented out.

Reply:
hm... the same is for example ADA and other chains
I think, this is chain specific and should stay in specific chain
for integration new one we will not change "constants" in core... do we?